skt/executable.py

```diff
@@ -222,33 +234,42 @@ def cmd_build(cfg):
     try:
         tgz = builder.mktgz(cfg.get('wipe'))
     except Exception as e:
-        save_state(cfg, {'buildlog': builder.buildlog})
+        save_state(
+            {
+                'buildlog': builder.buildlog
+            }
+        )
```

Comment:
Please avoid doing considerable reformatting together with logic changes. Please do the logic change in one commit, and reformatting in another, choose order as suits better. Then, reviews are easier, and we can drop or revert a problematic logic change, while still benefiting from reformatting.